src/data/SubSystems.js

```diff
@@ -0,0 +1,29 @@
+export const SubSystems = [
+    {
+        name: "Skirmish",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        code: "SKI",
+        gameSystemCode: "AOS",
+    },
+
+    {
+        name: "Skirmish",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        code: "SKI",
+        gameSystemCode: "AOF",
```

Comment:
We could factorize those two "Skirmish" by doing:
`gameSystemCode: ["AOS","AOF"],`

Reply:
Ok, done